### PR TITLE
And then there was only one (portal)

### LINKED ISSUE #3571: WordPress Patient Portals are really really broken
### Describe the bug

All WordPress up-for-grabs demos, including 5.0.2, are broken. A 500 error occurs when trying to login to the admin area.

tbh WordPress has been broken for awhile and we need to address it now. I just tried installing the WordPress patient portal and it's not at all updated.

@sunsetsystems seems to have done some work on this -- any help is appreciated

### PR DESCRIPTION
**Pull request by bradymiller**
fixes #3571

Just wanted to see what it would entail to remove the offsite and wordpress portals. Still gotta test this. And still gotta make sure community agrees with this.

### CODE REVIEW

File: interface/super/edit_globals.php [hunk 1]
```diff
@@ -801,16 +801,6 @@ function validate_file() {
                                             }
                                             ++$i;
                                         }
-
-                                        if (trim(strtolower($fldid)) == 'portal_offsite_address_patient_link' && !empty($GLOBALS['portal_offsite_enable']) && !empty($GLOBALS['portal_offsite_providerid'])) {
-                                            echo "<div class='row'>";
-                                            echo "<div class='col-sm-12'>";
-                                            echo "<input type='hidden' name='form_download' id='form_download'>";
-                                            echo "<button onclick=\"return validate_file()\" type='button'>" . xlt('Download Offsite Portal Connection Files') . "</button>";
```
Comment (bradymiller) [inline interface/super/edit_globals.php:809]: also gotta remove the validate_file() and other related code on this script

**Comment by bradymiller** [comment]
@tywrenn , figured you like seeing this PR :)

**Comment by bradymiller** [comment]
@tywrenn , very good point. Will make the change.

**Comment by sjpadgett** [comment]
I recommend for new name:
The Bestest Patient Portal
(For issues call 1-800-ASK-TYLER)

**Comment by bradymiller** [comment]
I think we should call the new portal: :drum: :drum: :drum: 
"Tyler's Place"

**Comment by bradymiller** [comment]
This is testing well and gonna bring it in very soon since there seems to be no resistance to the removal of the other portals :)

**Comment by sjpadgett** [comment]
Any ideas on what I need to add for WP replacement. I think one area deals with LBV forms. @sunsetsystems any thoughts of what my portal lacks that your portal covers?

**Comment by sunsetsystems** [comment]
@sjpadgett  I haven't looked at the portals in a while, but have you looked at this:
https://www.open-emr.org/wiki/index.php/Patient_Portal#CMS_Portal
In particular the section "Creating Forms in the CMS".

**Comment by bradymiller** [comment]
Gonna do it (bring this PR in). Next on the removal train is removing ccda mirth option and always using the local ccda. yes, it is raining Jerry (portal and ccda have taken OpenEMR by storm :) )

https://www.youtube.com/watch?v=BVDnDORSY9E

**Pull request merged**